doc/lua_api.txt

```diff
@@ -6911,16 +6925,63 @@ of the `${k}` syntax in formspecs is not deprecated.
 * `set_float(key, value)`
 * `get_float(key)`: Returns `0` if key not present.
 * `get_keys()`: returns a list of all keys in the metadata.
-* `to_table()`: returns `nil` or a table with keys:
-    * `fields`: key-value storage
-    * `inventory`: `{list1 = {}, ...}}` (NodeMetaRef only)
-* `from_table(nil or {})`
-    * Any non-table value will clear the metadata
-    * See [Node Metadata] for an example
-    * returns `true` on success
+* `to_table()`:
+    * Returns metadata as a metadata table (see below), if possible
+    * Returns `nil` on failure
```

Comment:
What does "if possible" refer to? When it it not possible? Proposed alternative:

```suggestion
    * Returns a Metadata table (see below) or `nil` on failure.
```